src/css/fonts.css

```diff
@@ -0,0 +1,51 @@
+@charset "UTF-8";
+
+@import url("https://fonts.googleapis.com/css?family=Source+Sans+Pro:300,300i,400,400i,600,600i,700,700i,900,900i");
+@import url("https://fonts.googleapis.com/css?family=Roboto+Slab:300,400,700");
+@import url("https://fonts.googleapis.com/css2?family=Source+Serif+Pro:ital,wght@0,300;0,400;0,600;0,700;0,900;1,200;1,300;1,400;1,600;1,700;1,900&display=swap");
+
+/** Import Font Awesome Fonts */
+@font-face {
+  font-family: 'Font Awesome 5 Free';
+  font-style: normal;
+  font-weight: 400;
+  font-display: auto;
+  src: url("~@fortawesome/fontawesome-free/webfonts/fa-regular-400.eot");
+  src: url("~@fortawesome/fontawesome-free/webfonts/fa-regular-400.eot?#iefix") format("embedded-opentype"),
+  url("~@fortawesome/fontawesome-free/webfonts/fa-regular-400.woff2") format("woff2"),
+  url("~@fortawesome/fontawesome-free/webfonts/fa-regular-400.woff") format("woff"),
+  url("~@fortawesome/fontawesome-free/webfonts/fa-regular-400.ttf") format("truetype"),
+  url("~@fortawesome/fontawesome-free/webfonts/fa-regular-400.svg#fontawesome") format("svg");
+}
+
+@font-face {
+  font-family: 'Font Awesome 5 Solid';
```

Comment:
Here's the weird thing. Both solids and regulars actually have the same font-family name "Font Awesome 5 Free" (like what you did for v6 here)
https://github.com/SU-SWS/decanter/blob/master/core/src/scss/core/fonts/_fontawesome-fonts.scss